src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1901,7 +1897,7 @@
         },
 
         "properties": {
-          "description": "Key/value pairs that provide additional information about the code location.",
+          "description": "Key/value pairs that provide additional information about the threadflow location.",
           "type": "object",
```

Comment:
>type": "object", [](start = 11, length = 16)

DRY this one out to `#/definitions/propertyBag`.

Reply:
Forgot to complete this scrub! Missed many examples, now resolved, thanks.

---
In reply to: [226731555](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226731555) [](ancestors = 226731555)

Reply:
Almost! Do the same for `ruleConfiguration.parameters`.

---
In reply to: [226844215](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226844215) [](ancestors = 226844215,226731555)

Reply:
Resolved in subsequent PR.

---
In reply to: [226873010](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226873010) [](ancestors = 226873010,226844215,226731555)